content/about/pricing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Pricing
+weight: 4
+---
+
+Workers is now free to enable for all and are subject to some limits. 
+
+## Unlimited 
+To avoid the limits and enable KV, you can subscribe to the Workers Unlimited Plan for **$5 USD per month for a account**. Workers Unlimited plan is separate from any Cloudflare plan (Free, Professional, Business) you may have.
```

Comment:
Do we need to specify that this is "for an account" or can we simply say that you can subscribe "for $5 USD per month" ? The latter seems simpler.

Also, consider starting the second sentence, "The Workers Unlimited plan", as that's how you referred to it in the previous sentence.

Consider "any _other_ Cloudflare plan".